src/test/java/com/google/cloud/resourcemanager/ResourceManagerImplTest.java

```diff
@@ -387,73 +438,210 @@ public void testTestPermissions() {
 
   @Test
   public void testRetryableException() {
-    ResourceManagerRpcFactory rpcFactoryMock = EasyMock.createMock(ResourceManagerRpcFactory.class);
-    ResourceManagerRpc resourceManagerRpcMock = EasyMock.createMock(ResourceManagerRpc.class);
-    EasyMock.expect(rpcFactoryMock.create(EasyMock.anyObject(ResourceManagerOptions.class)))
-        .andReturn(resourceManagerRpcMock);
-    EasyMock.replay(rpcFactoryMock);
-    ResourceManager resourceManagerMock =
-        ResourceManagerOptions.newBuilder()
-            .setServiceRpcFactory(rpcFactoryMock)
-            .build()
-            .getService();
-    EasyMock.expect(resourceManagerRpcMock.get(PARTIAL_PROJECT.getProjectId(), EMPTY_RPC_OPTIONS))
-        .andThrow(new ResourceManagerException(500, "Internal Error"))
-        .andReturn(PARTIAL_PROJECT.toPb());
-    EasyMock.replay(resourceManagerRpcMock);
-    Project returnedProject = resourceManagerMock.get(PARTIAL_PROJECT.getProjectId());
-    assertEquals(
-        new Project(resourceManagerMock, new ProjectInfo.BuilderImpl(PARTIAL_PROJECT)),
-        returnedProject);
+    String exceptionMessage = "Internal Error";
+    Mockito.doThrow(new ResourceManagerException(500, exceptionMessage))
+        .when(resourceManagerMock)
+        .get(PARTIAL_PROJECT.getProjectId());
+    try {
+      resourceManagerMock.get(PARTIAL_PROJECT.getProjectId());
+      fail();
+    } catch (RuntimeException expected) {
```

Comment:
Does this really throw a raw RuntimeException, not a subclass? If so, that probably needs to be fixed. If not, you can catch the subclass here.

Reply:
Even more importantly, what is this testing? As I read it, you configure a mock to throw an exception, and then test that the mock throws the exception. I don't think any actual model code is tested in any way.

Reply:
done